scripts/run_experiment.sh

```diff
@@ -38,29 +43,38 @@ cat << EOF > $GET_RESULTS_SH
 #SBATCH --time=00:00:59
 #SBATCH -A plgccbmc11-cpu
 
-SACCT_RESULT="\$(sacct -j $COLLECT_ID --format State,End)"
-echo \$SACCT_RESULT
-arrIN=(\${SACCT_RESULT//;/ })
-STATE=\$(echo \${arrIN[4]})
-if [[ "\$STATE" == "COMPLETED" ]]
-then
-    END=\$(echo \${arrIN[5]})
-    T_END=\$(date --date="\$END" +"%s%N")
-elif [[ "\$STATE" == "FAILED" ]]
-then
-    echo "Experiment failed, exiting..."
-    exit 1
-fi
+while true
+do
+    SACCT_RESULT="\$(sacct -j $COLLECT_ID --format State,End)"
```

Comment:
Ogólnie pytanie co do tej pętli:
Jest możliwe że ten skrypt się odpali kiedy status jest inny niż "completed"?

Bo ja przez "afterok" rozumiem, że to się ma odpalić jak dependencja się z sukcesem skończy

Reply:
No właśnie problem był w tym, że się odpalał, gdzie `SACCT ` zwracał, że `collect` dalej był w stanie `RUNNING` - wystawię issue z szerszym opisem

Reply:
ok